csharp/src/ProtocolBuffers/FieldCodec.cs

```diff
@@ -68,12 +68,12 @@ public static FieldCodec<int> ForSInt32(uint tag)
 
         public static FieldCodec<uint> ForFixed32(uint tag)
         {
-            return new FieldCodec<uint>(input => input.ReadFixed32(), (output, value) => output.WriteFixed32(value), CodedOutputStream.ComputeFixed32Size, tag);
+            return new FieldCodec<uint>(input => input.ReadFixed32(), (output, value) => output.WriteFixed32(value), 4, tag);
```

Comment:
I consider the magic numbers less readable and error prone that the previous version that clearly indicated what is the purpose of the number.